src/cpp/src/visual_language/inputs_embedder.cpp

```diff
@@ -1157,6 +1153,313 @@ class InputsEmbedderInternVLChat : public InputsEmbedder::IInputsEmbedder {
+    auto t25 = make_shared<Divide>(t16, t24, "numpy");
+    auto t26 = make_shared<Floor>(t25);
+    auto t27 = make_shared<Constant>(i32, Shape{}, vector<int32_t>{0});
+    auto t28 = make_shared<Unsqueeze>(t26, t27);
+    auto t29 = make_shared<Constant>(i64, Shape{1}, vector<int64_t>{2});
+    auto t30 = make_shared<Constant>(i64, Shape{1}, vector<int64_t>{2});
+    auto t31 = make_shared<Concat>(NodeVector{t8, t28, t29, t28, t30, t21}, 0);
+    auto t32 = make_shared<Reshape>(t23, t31, false);
+    auto t33 = make_shared<Constant>(i64, Shape{6}, vector<int64_t>{0, 1, 3, 2, 4, 5});
+    auto t34 = make_shared<Transpose>(t32, t33);
+    auto t35 = make_shared<Constant>(i64, Shape{1}, vector<int64_t>{-1});
+    auto t36 = make_shared<Constant>(i64, Shape{1}, vector<int64_t>{4});
+    auto t37 = make_shared<Multiply>(t21, t36, "numpy");
+    auto t38 = make_shared<Concat>(NodeVector{t8, t35, t37}, 0);
+    auto t39 = make_shared<Reshape>(t34, t38, false);
+    auto t40 = make_shared<Multiply>(t1, t2, "numpy");
+    auto t41 = make_shared<Convert>(t40, element::i64);
+    auto t42 = make_shared<Divide>(t6, t41, "numpy");
+    auto t43 = make_shared<Floor>(t42);
+    auto t44 = make_shared<Constant>(i64, Shape{}, vector<int64_t>{0});
+    auto t45 = make_shared<Unsqueeze>(t43, t44);
+    auto t46 = make_shared<Convert>(t1, element::i64);
+    auto t47 = make_shared<Unsqueeze>(t46, t44);
+    auto t48 = make_shared<Convert>(t2, element::i64);
+    auto t49 = make_shared<Unsqueeze>(t48, t44);
+    auto t50 = make_shared<Constant>(i64, Shape{1}, vector<int64_t>{-1});
+    auto t51 = make_shared<Concat>(NodeVector{t45, t47, t49, t28, t28, t50}, 0);
+    auto t52 = make_shared<Reshape>(t39, t51, false);
+    auto t53 = make_shared<Constant>(i64, Shape{6}, vector<int64_t>{0, 1, 3, 2, 4, 5});
+    auto t54 = make_shared<Transpose>(t52, t53);
+    auto t55 = make_shared<Multiply>(t1, t15, "numpy");
+    auto t56 = make_shared<Convert>(t55, element::i64);
+    auto t57 = make_shared<Constant>(i64, Shape{}, vector<int64_t>{2});
+    auto t58 = make_shared<Divide>(t56, t57, "numpy");
+    auto t59 = make_shared<Floor>(t58);
+    auto t60 = make_shared<Constant>(i32, Shape{}, vector<int32_t>{0});
+    auto t61 = make_shared<Unsqueeze>(t59, t60);
+    auto t62 = make_shared<Multiply>(t2, t15, "numpy");
+    auto t63 = make_shared<Convert>(t62, element::i64);
+    auto t64 = make_shared<Constant>(i64, Shape{}, vector<int64_t>{2});
+    auto t65 = make_shared<Divide>(t63, t64, "numpy");
+    auto t66 = make_shared<Floor>(t65);
+    auto t67 = make_shared<Unsqueeze>(t66, t60);
+    auto t68 = make_shared<Concat>(NodeVector{t45, t61, t67, t37}, 0);
+    auto t69 = make_shared<Reshape>(t54, t68, false);
+    shared_ptr<Model> model = make_shared<Model>(make_shared<Result>(t69), ParameterVector{t0, t1, t2});
+    return utils::singleton_core().compile_model(
+        model, "CPU"
+    ).create_infer_request();
```

Comment:
how to maintain it?

Reply:
What's so special about it?

Reply:
is it auto-generated?

Reply:
in this case you need to add steps how to "reproduce" this code.

Reply:
Added